lib/options.js

```diff
@@ -50,6 +50,11 @@ exports.options = [
     name: 'treeless',
     description: `Don't display statics tree`,
     defaultValue: false
+  },
+  {
+    name: 'ssl',
+    description: 'Serve with ssl',
```

Comment:
> Serve content using SSL

Reply:
Sounds good; wasn't sure what the ideal copy was so I figured someone would eventually chime in :wink: